src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -41,12 +40,12 @@ def validate_association(
     diff = [str(item) for item in raw_items if item not in sentry_items]
     unique_diff = list(dict.fromkeys(diff).keys())
 
-    if len(unique_diff):
-        return [
-            f'The following {type} do not have an association in Sentry: {", ".join(unique_diff)}.'
-        ]
+    return unique_diff if len(unique_diff) else []
+    #     return [
```

Comment:
It's better to delete the code than comment it out because code inside comments is too difficult to maintain.

Reply:
@mgaeta nah I should rm this. It's not needed.